### PR TITLE
Update the title

### PR DESCRIPTION
**Pull request by IMBLAH**

### CODE REVIEW

File: _posts/IMBLAH/2021-11-10-IMBLAH-project-idea.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 author: IMBLAH
-title: “Bo's Project idea & work plan”
+title: Bo's Project idea & work plan
```
Comment (MishaCienfuegos) [inline _posts/IMBLAH/2021-11-10-IMBLAH-project-idea.md:4]: Thanks for removing these curly quotes `“`; other students have had unanticipated problems using them in their projects and posts. However, it is safest to enclose this title string in straight quotes `"Bo's Project idea & work plan"`.

**Review by MishaCienfuegos — APPROVED** [review]
This is looking good! I had one small suggestion to make sure your title is treated as a string.

**Review by eah13 — CHANGES_REQUESTED** [review]
You'll need `"` around your title, because the apostrophe will confuse our class website otherwise.

**Comment by eah13** [comment]
fixed by #613, so closing this PR

**Pull request closed without merging**